instance/tasks.py

```diff
@@ -73,6 +73,14 @@ def watch_pr():
                             .format(pr=pr, i=instance, truncated_title=truncated_title)
             instance.github_pr_url = pr.github_pr_url
             instance.ansible_extra_settings = pr.extra_settings
+            instance.ansible_source_repo_url = pr.extra_settings_dict.get('edx_ansible_source_repo')
+            if not instance.ansible_source_repo_url:
+                instance.ansible_source_repo_url = 'https://github.com/{}.git'.format(
+                    settings.DEFAULT_CONFIGURATION_FORK
+                )
+            instance.configuration_version = pr.extra_settings_dict.get('configuration_version')
+            if not instance.configuration_version:
+                instance.configuration_version = settings.DEFAULT_CONFIGURATION_VERSION
```

Comment:
I believe this code needs to be here to force fields to the new defaults when creating instances from PRs (unless values are explicitly provided in the PR body).

However, as mentioned above, I would prefer if the default was also set in a pre-save hook so that the value in the db would never be blank for these fields.